### DISCUSSION TITLE
A small Mod delay movement “glitch”.

### DISCUSSION THREAD

**Discussion by tripsylon** [bangcorrupt/hacktribe#295] [Q&A] [answered]
First of all thank you Bangcorrupt for the great Mod Delay🙏🙂. But I’ve noticed that when you use the mod delay, and record mfx movement, that after clearing the movement you can’t use the mod delay until you switch to another mfx type and back… Which isn’t a total deal breaker , but is a bit anoing to me because the Mod Delay is really superb and I use it all the time it’s the best Mfx on the machine 😌👌👌👌

**Comment by bangcorrupt** [comment, marked as answer]
I can't reproduce this, can you share detailed steps?

I tried:
1.  Initialise pattern (Mod Delay is default MFX)
2. Write pattern
3. Play pattern
4. Record MFX motion
5. Write pattern
6. Clear MFX motion
7. Write pattern

Result:

MFX Mod Delay still works, including MFX motion recording.

I also tried with Tap Delay Sync MFX and not writing the pattern at each step. I found no issues.

Do you have the issue if you start from an initialised pattern?

Can you share a minimal pattern file that demonstrates the issue?

**Reply by tripsylon** [reply]
First of all I want to apologize, because I was speaking about the TAP DELAY SYNC all the time 😓 it’s really, really great 👌… And  about the problem , after switching the machine on and off everything is working fine  in the affected and all other patterns… it only happened two or three times in the past and like I said after switching to other mfx types the recording went back to normal… next time I only report problems that are reproducible and happen more frequently… Sorry for the the alert .

**Reply by bangcorrupt** [reply]
No problem, keep an eye out for it.  It may only occur under rare circumstances.
